### ISSUE TITLE
Query param for Url 

### ISSUE THREAD

**Issue by mijaelcohen** [lucasconstantino/graphiql-online#8]
this project is awesome but its tedius to add the url by hand each time I use it.
would be great to be a query parameter (i got this in my bookmark)

**Comment by iwconfig** [comment]
Yep, really annoying actually. It's a shame #1 never got merged.

**Comment by iwconfig** [comment]
I modified `graphiql-online.js` in `gh-pages` branch to work with url search params.

I don't know *anything* about React so maybe I missed something or if it's supposed to generate the `graphiql-online.js` file or whatever so I'm leaving the changes in my own fork for now instead of issuing a PR to upstream.

https://iwconfig.github.io/graphiql-online/?endpoint=https://whatever

## update

- Also added `query` and `variables` parameters,
- and now `endpoint` is saved to localStorage as well.
